runtime/openwhisk/Cargo.toml

```diff
@@ -9,9 +9,9 @@ serde = { version = "1.0.81", features = ["derive"]}
 serde_json = "1.0.81"
 serde_derive = "1.0.81"
 anyhow = "1.0.56"
-wasmtime = "0.34"
-wasmtime-wasi = "0.34"
-wasi-common = "0.34"
+wasmtime = "0.36.0"
```

Comment:
Different version of same crates are seen. revert the unnecessary version update